manala.apt/vars/main.yml

```diff
@@ -427,9 +427,10 @@ manala_apt_keys_patterns:
     }[ansible_distribution|lower]
   }}"
   # Deprecated
-  sury_php_debian:
+  
+  _php_debian:
```

Comment:
Not sure this yml key has to be changed

Reply:
Even if it's legacy, it's still concerned by this key change I think since it's debian related. But I might be wrong.

Reply:
HUM. not sure why this key has changed. It wasn't my intention to do so.... 🤔 good catch.